packages/sdk/src/waku.ts

```diff
@@ -180,6 +184,19 @@ export class WakuNode implements Waku {
     await this.libp2p.stop();
   }
 
+  async subscribeToContentTopic(
+    contentTopic: string,
+    peer: Multiaddr,
+    callback: Callback<DecodedMessage>
+  ): Promise<IFilterSubscription> {
+    return (
+      await subscribeToContentTopic(contentTopic, callback, {
+        waku: this as LightNode,
+        peer
+      })
+    )[0];
```

Comment:
nit: let's not do `[0]` access as it is not safe

Reply:
updated in latest commit